### PR TITLE
YarrrDrobe

### PR DESCRIPTION
**Pull request by Salvantrix**
Adds the new pirate drobe vending machine for all your pirate wears n such.

## Why / Balance

Pirates need clothes too!

## How to test

Open game, place Drobe, profit.

## Media

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase
- 
image

## Breaking changes

**Changelog**

Adds the YarrrDrobe
<!--

### CODE REVIEW

**Comment by Salvantrix** [comment]
YML errors will clear once uplink and sprites are merged just FYI

File: Resources/Locale/en-US/advertisements/vending/yarrrdrobe.ftl [hunk 1]
```diff
@@ -0,0 +1,5 @@
+advertisement-yarrrdrobe-1 = Eye patches 50% off.
+advertisement-yarrrdrobe-2 = Plunder some booty while you dress as a beauty!
+advertisement-yarrrdrobe-3 = Who needs an extra leg anyways?
+advertisement-yarrrdrobe-4 = Come get yer supplies while you last!
+advertisement-yarrrdrobe-5 = Oi! Land lubber get o'er here and plunder me!
```
Comment (whatston3) [inline Resources/Locale/en-US/advertisements/vending/yarrrdrobe.ftl:5]: Small rewrite.
```suggestion
advertisement-yarrrdrobe-5 = Oi, landlubber! Get o'er here and plunder me!
```

File: Resources/Prototypes/_NF/Catalog/VendingMachines/Inventories/yarrrdrobe.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+- type: vendingMachineInventory
+  id: YarrrDrobeInventory
+  startingInventory:
+    ClothingNeckCloakPirateCap: 4
+    ClothingNeckCloakPirateParrot: 4
+    ClothingShoesBootsMagPirateFilled: 4
```
Comment (whatston3) [inline Resources/Prototypes/_NF/Catalog/VendingMachines/Inventories/yarrrdrobe.yml:6]: A few questions here -
1. Does a pirate have access to a bank account?  If so, would it be reasonable to assume they can purchase this?
2. Why would I pay my precious doubloons for magboots (without a knife!) when I could pay spesos for the same thing and use the doubloons for, say, extra cannonballs?

File: Resources/Locale/en-US/advertisements/vending/yarrrdrobe.ftl (hunk 1 above)
Comment (whatston3) [reply Resources/Locale/en-US/advertisements/vending/yarrrdrobe.ftl:5]: Other suggestions:
X marks the spot for incredible deals!
Best outfits this side of Tortuga.
Brace the main sale!

**Review by whatston3 — DISMISSED** [review]
Tested by pulling in the head of Pirate-Stuff locally.  Looks great!  Left a few questions & suggestions for your consideration, looking forward to seeing it in-game.

**Comment by Salvantrix** [comment]
> Tested by pulling in the head of Pirate-Stuff locally.  Looks great!  Left a few questions & suggestions for your consideration, looking forward to seeing it in-game.

Np! Also to answer pirates do have bank accounts I think? And with this change being a push to round start slots openings shouldn't be an issue

**Review by Cheackraze — APPROVED** [review]

**Pull request merged**